apps/spruce/src/pages/waterfall/InactiveVersionsButton/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import Button from "@leafygreen-ui/button";
+import { palette } from "@leafygreen-ui/palette";
+import pluralize from "pluralize";
+import { DisplayModal } from "components/DisplayModal";
+import Icon from "components/Icon";
+import { size } from "constants/tokens";
+import { WaterfallQuery } from "gql/generated/types";
+import { VersionLabel } from "../VersionLabel";
+
+const { gray } = palette;
+interface Props {
+  versions: WaterfallQuery["waterfall"]["versions"][0]["inactiveVersions"];
+  containerHeight: number | undefined;
+}
+export const InactiveVersionsButton: React.FC<Props> = ({
+  containerHeight,
+  versions,
+}) => {
+  const [modalOpen, setModalOpen] = useState(false);
+  return (
+    <>
+      <DisplayModal
+        data-cy="inactive-versions-modal"
+        open={modalOpen}
+        setOpen={setModalOpen}
+        title={`${versions?.length} Inactive ${pluralize("Version", versions?.length)}`}
```

Comment:
This text may change when DEVPROD-11706 is implemented. "inactive-versions" may change to "inactive-and-filtered" depending on how that ticket is implemented.